tempesta_fw/main.c

```diff
@@ -341,7 +341,7 @@ static struct ctl_table tfw_sysctl_tbl[] = {
 		.mode		= 0644,
 		.proc_handler	= tfw_ctlfn_state_io,
 	},
-	{ 0 }
+	{}
```

Comment:
Although GCC complies the code, the statement is valid for C++ and prohibited for C.  `{ 0 }` is more correct for C.